src/overcooked_demo/server/static/templates/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
```

Comment:
Plus these html files

Reply:
Sorry I am not sure if I understand? Are you talking about a particular line or the entire file? I haven't checked all of the js and html files, but at very least the index.js and index.html are definitely required since they are what the browser render and what defines the UI and interactions. 

Are there reasons why you think these aren't required?

Reply:
Was talking about the HTML files! I guess they didn't seem immediately relevant, and this might be a good time to clean some unused files up!

Reply:
All the html files are still in use except the psiturk one, for the same reason as the psiturk.js file. All the html defines how the webpages look like, for instance they help define where the game window is and where each button is, even though the game logics reside in other files.